frontend/src/App.svelte

```diff
@@ -26,11 +27,13 @@
   let selected_node = null;
 
   let show_menu = false;
-  let show_legend = false;
+  let show_legend = true;
   let show_timebar = false;
 
-  let legend = new Map<string, string>();
-  $: colors = [...legend.entries()];
+  let customTheme = config.Theme.ColorBlind;
+  let themeMap = new Map(Object.entries(customTheme));
+  let legend = themeMap;
```

Comment:
Missing type declarations

Reply:
Was declared this way originally. Was afraid of adding too much changes.